scripts/word_embeddings/evaluate_pretrained.py

```diff
@@ -150,9 +149,11 @@ def load_embedding_from_path(args):
         if args.max_vocab_size:
             idx_to_token = idx_to_token[:args.max_vocab_size]
 
-        with utils.print_time('compute vectors from subwords '
-                              'for {} words.'.format(len(idx_to_token))):
-            embedding[idx_to_token] = model[idx_to_token]
+        # If there are any remaining tokens we may precompute
+        if idx_to_token:
+            with utils.print_time('compute vectors from subwords '
```

Comment:
is this util only available for embedding training? It looks very useful and should probably be added to gluonnlp api

Reply:
It's only in the scripts folder. It's very simple, but I agree may be useful to add to main API

```
@contextmanager
def print_time(task):
    start_time = time.time()
    logging.info('Starting to %s', task)
    yield
    logging.info('Finished to {} in {:.2f} seconds'.format(
        task,
        time.time() - start_time))

```

Do you have any suggestions on the logging format used here?